packages/integration-tests/suites/tracing/browsertracing/pageload-meta/test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getSentryTransactionRequest } from '../../../../utils/helpers';
+
+sentryTest(
+  'should create a pageload transaction based on `sentry-trace` <meta>',
+  async ({ getLocalTestPath, page }) => {
+    const url = await getLocalTestPath({ testDir: __dirname });
+
+    const eventData = await getSentryTransactionRequest(page, url);
+
+    expect(eventData.contexts?.trace).toMatchObject({
+      op: 'pageload',
+      parent_span_id: '1121201211212012',
+      trace_id: '12312012123120121231201212312012',
+    });
+
+    expect(eventData.spans?.length).toBeGreaterThan(0);
```

Comment:
Can we assert that navigation transactions won't have the same trace ID and parent span ID?

Reply:
👍  adding the assertions to navigation tests at -> `packages/integration-tests/suites/tracing/browsertracing/navigation/test.ts`

Reply:
This is also needed for the meta tests, to make sure that navigation transactions don't continue a transaction from the meta header.